azurerm/resource_arm_iothub.go

```diff
@@ -313,6 +362,11 @@ func resourceArmIotHubCreateUpdate(d *schema.ResourceData, meta interface{}) err
 		Tags: expandTags(tags),
 	}
 
+	if !d.IsNewResource() {
```

Comment:
I don't really see the point of this diff.
When I removed it and set the `Properties.Routing.FallbackRoute` property to `fallbackRoute` in the line above, fallback routes started working